src/item.cpp

```diff
@@ -2285,8 +2285,8 @@ int item::get_storage() const
     int result = (static_cast<int> (static_cast<unsigned int>( t->storage ) ) );
 
     if (item::item_tags.count("pocketed") > 0){
-        pockets = ((volume() * get_coverage()) / 100) / 1.333;
-        if (result > (volume() / 2)){
```

Comment:
Could I suggest a more gradual scaling then? The big problem I'm seeing with this is it creates a very sharp and very drastic change when the object's natural storage is one half of the object's volume. Consider this:
```
Armor 1 fur version:
20 storage
40 volume
100% coverage
Storage gained: 40 / 1.333 = 30
Total storage w/ mod: 50

Armor 1 normal version (-2 volume)
20 storage
38 volume
Storage gained: 38 / 1.333 / 2 = 14
Total storage w/ mod: 34
```
The basic numbers are totally made up in this case of course, but we have armor that lands on both sides of the cut, and something that lands across it is totally possible under the current system.

Something like this would produce a much more gradual curve:
```c++
if (result > .3 * volume()) {
    float mod = 1 + (result - .3 * volume()) / (.7 * volume() - .3 * volume()) * 2;
    pockets = pockets / mod;
}
```
If we run the same armors through the new calculation we get:
```
Armor 1
Gained storage:15
Total storage:35

Armor 2
Gained storage:13
Total storage:33